mdagent/tools/maketools.py

```diff
@@ -65,6 +67,8 @@ def make_all_tools(
         RDFTool(path_registry=path_instance),
         SetUpandRunFunction(path_registry=path_instance),
         SimulationOutputFigures(path_registry=path_instance),
+        ContactsTool(path_registry=path_instance),
+        DistanceMatrixTool(path_registry=path_instance),
```

Comment:
put them in alphabetical order here as well -- our list is getting longer and longer, so organize them will make things easier to find & edit : )